test/rules-google-enhanced-ecommerce-fullstory.spec.ts

```diff
@@ -0,0 +1,343 @@
+import 'mocha';
+
+import { expectEqual } from './utils/mocha';
+import { RulesetTestHarness, getRulesetTestEnvironments } from './utils/ruleset-test-harness';
+
+// TODO(nate): Update this ruleset to match the rules served by FullStory data layer capture integrations.
+// This will require supplying a "convert" beforeDestination operator to convert string currency values
+// to numbers.
+import '../rulesets/google-ua-enhanced-ecommerce.js';
```

Comment:
I opted not to update the Enhanced Ecommerce ruleset to match the ruleset served with FullStory data layer capture integrations in this PR. This ruleset is more complicated, and the changes cut deeper and would've resulted in a larger PR surface. I'm hoping to true up this ruleset in a future dedicated PR.